recipe/meta.yaml

```diff
@@ -24,10 +24,12 @@ requirements:
     - python
     - pip
     - snappy
-    - msinttypes  # [win and py27]
+    - msinttypes      # [win and py27]
```

Comment:
This can be dropped. We no longer support Python 2.7 in conda-forge. Also dropping this will fix the linter error.

```suggestion
```